packages/custom_lint_builder/lib/src/client.dart

```diff
@@ -330,6 +333,7 @@ class _ClientAnalyzerPlugin extends analyzer_plugin.ServerPlugin {
     required super.resourceProvider,
   });
 
+  PackageConfig? _packageConfig;
```

Comment:
Don't store it in a local variable. Instead pass it to _CustomLintAnalysisConfigs.from

Otherwise LGTM

Reply:
Done!